ethereum/networks/src/main/java/tech/pegasys/teku/networks/Eth2NetworkConfiguration.java

```diff
@@ -423,11 +428,6 @@ public Eth2NetworkConfiguration build() {
                         if (maybeEpochsStoreBlobs.isPresent()) {
                           denebBuilder.epochsStoreBlobs(maybeEpochsStoreBlobs);
                         }
-                        if (trustedSetup.isEmpty()) {
-                          LOG.warn(
-                              "Setting a default for trusted setup as nothing was set explicitly");
-                          trustedSetupFromClasspath(MAINNET_TRUSTED_SETUP_FILENAME);
-                        }
```

Comment:
I think we shouldn't be altering this part in our change. can we restore this please?

Reply:
I am not sure what is been altered here.

Reply:
I am not sure what is been altered here